block-storage/src/main/scala/coop/rchain/blockstorage/ApprovedStore.scala

```diff
@@ -8,10 +8,10 @@ import coop.rchain.shared.syntax._
 import coop.rchain.store.{KeyValueStoreManager, KeyValueTypedStore}
 import scodec.codecs._
 
-object ApprovedStore {
+object approvedStore {
```

Comment:
I'm not sure this is a good change, per Scala [naming conventions](https://docs.scala-lang.org/style/naming-conventions.html#objects) object name should start with uppercase with rare exceptions

Reply:
Fixed in https://github.com/rchain/rchain/pull/3673/commits/97886a864d6cf0329c539a83f1ed053969e53d24 for block store. Approved block will be removed for HF2 anyway.